src/components/Image.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable complexity */
```

Comment:
we never put this at the very top or am i wrong? it should be placed right infront of the code, that forces the linting issue

Reply:
Yes. At the same time, I reactivated the function where it had ended.
https://github.com/smart-village-solutions/smart-village-app-app/pull/957/commits/4879aaf1097c8856e567a9ae1ffd90d32d17d8cc